src/Tezos/Forging.cpp

```diff
@@ -125,11 +179,127 @@ Data forgeOperation(const Operation& operation) {
         append(forged, forgedGasLimit);
         append(forged, forgedStorageLimit);
         append(forged, forgedAmount);
-        append(forged, forgeBool(false));
```

Comment:
I wonder why this forgeBool was here in the first place @hewigovens 

according to: https://tezos.gitlab.io/shell/p2p_api.html#transaction-tag-108 there is no boolean before destination